src/map/script.c

```diff
@@ -25596,6 +25596,25 @@ static BUILDIN(identifyidx)
 	return true;
 }
 
+static BUILDIN(openlapineddukddakboxui)
```

Comment:
```suggestion
static BUILDIN(openlapineddukddak)
```
```suggestion
static BUILDIN(lapineddukddak)
```
shorter? since everyone know it will open the UI.

Reply:
To keep consistent with the refinery ui command and considering that there is another type of lapine ui with similar name i'd rather keep it as is.